include/nbl/builtin/hlsl/random/xoroshiro.hlsl

```diff
@@ -0,0 +1,47 @@
+
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_BUILTIN_GLSL_RANDOM_XOROSHIRO_HLSL_INCLUDED_
+#define _NBL_BUILTIN_GLSL_RANDOM_XOROSHIRO_HLSL_INCLUDED_
+
+#include <nbl/builtin/hlsl/math/functions.hlsl>
+
+
+namespace nbl
+{
+namespace hlsl
+{
+
+#define xoroshiro64star_state_t uint2
+#define xoroshiro64starstar_state_t uint2
```

Comment:
Why do we need to `#define` here and not just use a `uint2` where necessary? The `xoroshiro64star_state_t` wasn't even used